packages/components/Notification/src/Notification.tsx

```diff
@@ -46,11 +48,12 @@ export const Notification = compose<NotificationType>({
     }, ['isBar']);
 
     return (final: NotificationProps, ...children: React.ReactNode[]) => {
-      const { variant, title, action, ...rest } = mergeProps(userProps, final);
+      const { variant, icon, title, action, ...rest } = mergeProps(userProps, final);
       const mergedProps = mergeProps<PressableProps>(rest, rootStyle);
 
       return (
         <Slots.root {...mergedProps}>
+          {icon && <Slots.icon {...icon} width={24} height={24}></Slots.icon>}
```

Comment:
Move the hardcoded size to icon's buildProps?

Reply:
Also, you didn't need to call `createIconProps` ?

Reply:
Nope, works without it which makes me think this code is buggy

Reply:
nvm, figured it out